airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/ConnectionTransformationTab/DbtCloudTransformationsCard.tsx

```diff
@@ -0,0 +1,181 @@
+      job: number().required().positive().integer(),
+    })
+  ),
+});
+
+export const DbtCloudTransformationsCard = ({ connection }: { connection: WebBackendConnectionRead }) => {
+  // Possible render paths:
+  // 1) IF the workspace has no dbt cloud account linked
+  //    THEN show "go to your settings to connect your dbt Cloud Account" text
+  //    and the "Don't have a dbt account?" hero/media element
+  // 2) IF the workspace has a dbt cloud account linked...
+  //   2.1) AND the connection has no saved dbt jobs (cf: operations)
+  //        THEN show empty jobs list and the "+ Add transformation" button
+  //   2.2) AND the connection has saved dbt jobs
+  //        THEN show the jobs list and the "+ Add transformation" button
+
+  const { hasDbtIntegration, saveJobs, dbtCloudJobs } = useDbtIntegration(connection);
+  const onSubmit = (values: DbtJobListValues, { resetForm }: FormikHelpers<DbtJobListValues>) => {
+    saveJobs(values.jobs).then(() => resetForm({ values }));
+  };
+
+  return (
+    <Formik // TODO extract to parent component, see if that helps with input focus issues
+      onSubmit={onSubmit}
+      initialValues={{ jobs: dbtCloudJobs }}
+      validationSchema={dbtCloudJobListSchema}
+      render={({ values, isValid, dirty }) => {
+        return (
+          <Form className={styles.jobListForm}>
+            <FormChangeTracker changed={dirty} />
+            <FieldArray
+              name="jobs"
+              render={({ remove, push }) => {
+                return (
+                  <Card
+                    title={
+                      <span className={styles.jobListTitle}>
+                        Transformations
+                        {hasDbtIntegration && (
+                          <Button
+                            variant="secondary"
+                            onClick={() => push({ account: "", job: "" })}
+                            icon={<FontAwesomeIcon icon={faPlus} />}
+                          >
+                            Add transformation
+                          </Button>
+                        )}
+                      </span>
+                    }
+                  >
+                    {hasDbtIntegration ? (
+                      <DbtJobsList jobs={values.jobs} remove={remove} isValid={isValid} dirty={dirty} />
+                    ) : (
+                      <NoDbtIntegration className={styles.jobListContainer} />
+                    )}
+                  </Card>
+                );
+              }}
+            />
+          </Form>
+        );
+      }}
+    />
+  );
+};
+
+const DbtJobsList = ({
+  jobs,
+  remove,
+  isValid,
+  dirty,
+}: {
+  jobs: DbtCloudJob[];
+  remove: (i: number) => void;
+  isValid: boolean;
+  dirty: boolean;
+}) => (
+  <div className={classNames(styles.jobListContainer, styles.emptyListContent)}>
+    <p className={styles.contextExplanation}>After an Airbyte sync job has completed, the following jobs will run</p>
```

Comment:
I think in case there are no jobs setup yet, we should not show this message, since we know there are no "following jobs". Maybe the empty state could be simply the image with the label below stating something along the lines of:

"No Transformations. You can add transformations to run when sync jobs have completed."